lib/widgets/search_widget.dart

```diff
@@ -23,7 +23,9 @@ class _SearchWidgetState extends ConsumerState<SearchWidget> {
         await ref.read(appsController.notifier).getFilteredConnectedApps(value);
         break;
       case SearchWidgetFilter.atSigns:
-        await ref.read(connectedAtsignsControllerProvider.notifier).getFilteredConnectedAtsign(value);
+        await ref
```

Comment:
There should be no changes to the search_widget. That's outside the scope of this PR.